### PR TITLE
Update to Fabric Loader 0.12.12, use Mixin 0.8.4

### PR DESCRIPTION
**Pull request by IMS212**
This allows us to get rid of 2 overwrites in favor of ModifyVariable.

### CODE REVIEW

File: src/main/resources/fabric.mod.json [hunk 1]
```diff
@@ -45,7 +45,7 @@
   "accessWidener" : "iris.accesswidener",
 
   "depends": {
-    "fabricloader": ">=0.7.4",
+    "fabricloader": ">=0.12.3",
```
Comment (DragonEggBedrockBreaking) [inline src/main/resources/fabric.mod.json:48]:
```suggestion
    "fabricloader": ">=0.12.32”,
```

Comment (DragonEggBedrockBreaking) [reply src/main/resources/fabric.mod.json:48]: Idk why GitHub formatted it so weirdly…

Comment (IMS212) [reply src/main/resources/fabric.mod.json:48]: Are you sure 0.12.2 works for this?

Comment (DragonEggBedrockBreaking) [reply src/main/resources/fabric.mod.json:48]: Oops

**Review by coderbot16 — APPROVED** [review]
Thanks. I'd like to replace the relevant Mixin entirely at some point but newer Mixin is nice. If the new Fabric Loader version requirement turns out to be an issue for modpacks we can always downgrade later.

**Pull request merged**